### PR TITLE
create method for check one nft collection

### PR DESCRIPTION
**Pull request by zakhar-petukhov**

### CODE REVIEW

File: pkg/api/nft_handlers.go [hunk 1]
```diff
@@ -78,3 +79,28 @@ func (h Handler) GetNftCollections(ctx context.Context, params oas.GetNftCollect
...
+	collection, err := h.storage.GetNftCollectionByCollectionAddress(ctx, account)
+	if errors.Is(err, core.ErrEntityNotFound) {
+		return &oas.NotFound{Error: err.Error()}, nil
+	}
+	if err != nil {
+		return &oas.InternalError{Error: err.Error()}, nil
+	}
+	col := convertNftCollection(collection)
+	if collection.Metadata != nil {
+		var metadata oas.OptNftCollectionMetadata
```
Comment (aleksej-paschenko) [inline pkg/api/nft_handlers.go:98]: would it make sense to handle the metadata assignment inside `convertNftCollection`?

**Review by aleksej-paschenko — APPROVED** [review]
looks good

File: pkg/api/nft_handlers.go (hunk 1 above)
Comment (zakhar-petukhov) [reply pkg/api/nft_handlers.go:98]: Great point 👌

Comment (zakhar-petukhov) [reply pkg/api/nft_handlers.go:98]: I'm going to change it

File: pkg/config/config.go [hunk 2]
```diff
@@ -25,6 +25,11 @@ type Config struct {
 	}
 }
 
+var Loader struct {
+	IpfsGate       string `env:"IPFS_GATE" envDefault:"https://ipfs.io/ipfs/"`
+	TonStorageGate string `env:"TONSTORAGE_GATE" envDefault:"http://storage.ton/gateway/"`
```
Comment (mr-tron) [inline pkg/config/config.go:30]: great defaults which don't work :)

**Pull request merged**